jwst/assign_wcs/tests/test_niriss.py

```diff
@@ -125,12 +125,14 @@ def traverse_wfss_trace(filtername):
     assert orderdet == orderin
 
 
+@pytest.mark.xfail(reason="New specwcs reference files are needed in CRDS")
```

Comment:
Is this still true or are the files recently delivered by Swara sufficient?

Reply:
removed